app/controllers/landing_page_controller.rb

```diff
@@ -36,7 +42,7 @@ def index
       if cache_meta.nil?
         cache_hit = false
         content = build_html(
-          community_id: cid,
+          community: community,
```

Comment:
We're trying to get rid of community etc. objects in 99,99% of the code. That's why in this controller community is only handled at the top level actions and from there on data is passed down as data. This change goes against that idea so I'd vote for passing api key directly.